src/components/marker/timeline-marker.tsx

```diff
@@ -62,6 +72,7 @@ export const TimelineMarker = (({isDisabled, onMarkerClick, getSeekBarNode, useQ
           onMouseOver={onMouseOver}
           tabIndex={disabled ? -1 : 0}
           data-testid="cuePointMarkerContainer"
+          aria-label={type}
```

Comment:
so with this aria label configured the screen reader does read it out ?

Reply:
yes, currently screen reader reads `unlabelled button`, after changes it reads f.e. `hotspot button`